execution/engine/emr_engine.go

```diff
@@ -207,10 +206,7 @@ func (emr *EMRExecutionEngine) driverPodTemplate(executable state.Executable, ru
 		workingDir = *run.SparkExtension.SparkSubmitJobDriver.WorkingDir
 	}
 
-	labels := map[string]string{
```

Comment:
could we add these to the default labels? Since this would help keep it backward compatible.

Reply:
done!